decorator.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'nameable'
+
```

Comment:
We need a constructor for nameable ie, `attr_accessor :nameable` Let's kindly fix :wrench: it